project/V.scala

```diff
@@ -72,6 +72,8 @@ object V {
     "2.13.2",
     "2.13.3",
     "2.13.4",
+    // has a binary compatibility issue
+    "2.13.9",
```

Comment:
It'll good to somehow inform users about that binary compatibility issue

Reply:
We can do it in another PR

Reply:
> we'll deprecate it separately

Do you mean, we'll drop 2.13.9 support from Metals? I think showing some kind of warning is enough.
(anyway, we can do that in another PR :+1:)